.github/templates/node-setup/action.yml

```diff
@@ -13,6 +13,10 @@ runs:
         node-version-file: .node-version
         cache: pnpm
 
+    - name: Keep npm up-to-date
```

Comment:
Should we add a small comment why this is needed so we are aware of it in the future?